next-app/src/components/views/Main/Main.style.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components'
+import { getTypographyStyles } from 'styles/fonts'
+import { colors } from 'styles/colors'
+
+export const Wrapper = styled.div`
+  margin: 100px 0;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 100px;
+`
+export const Container = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  /* border: 1px solid red; */
+`
+
+export const ContentsWrapper = styled.div`
+  margin: 0 180px;
+  width: 100%;
+  height: 400px;
+  /* border: 1px solid blue; */
+`
+export const Contents = styled.div`
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  gap: 80px;
+`
+
+export const ImageWrapper = styled.div`
+  width: 550px;
+  height: 100%;
+  border-radius: 12px;
+  background-color: ${colors.gray200};
+`
+export const BoardWrapper = styled.div`
+  width: 550px;
+  height: 100%;
+  display: flex;
+  align-items: center;
```

Comment:
```suggestion
  flex-direction: column;
  justify-content: center;
```
align-items 속성을 제거하고 위의 두 속성을 추가하면 S.BoardWrapper 바로 내부에 한번 더 감싸고 있는 <div>태그가 불필요 할 것 같아요 🥺